nextflow.config

```diff
@@ -52,6 +52,7 @@ params {
 
     // Variant Calling
     only_paired_variant_calling = false // if true, skips germline variant calling for normal-paired samples
+    joint_mutect2               = false // if true, enables patient-wise multi-sample somatic variant calling
```

Comment:
Can it be put after the joint_germline params ?

Reply:
Done!